### DISCUSSION TITLE
各分 P 间切换、同一合集内视频间切换不立即播放

### DISCUSSION THREAD

**Discussion by Geeyun-JY3** [the1812/Bilibili-Evolved#4499] [功能建议 / Ideas]
指的是通过任意方式切换（不过好像也包含了放完一个连播下一个这种切换模式，感觉好像会和[`传统连播模式`](https://github.com/the1812/Bilibili-Evolved/blob/master/doc/features/features.md#传统连播模式)冲突，可能得考虑一下怎么设计不冲突）
有这个需求主要还是有时只是想看一下合集内另一视频的封面或评论区，但在破站播放页界面不能通过上下文菜单等相关方法在另一标签页打开合集内其他视频，直接点击切换马上就播放了，有时也不想离开当前视频播放页面。
多 P 视频是顺带的（

***

不得不说这需求是有点怪（

**Comment by the1812** [comment]
这主要是 b 站的分 P 按钮不是 `<a>` 的缘故, 强行做这种功能出来也是播放一小会再暂停, 不治本

**Reply by Geeyun-JY3** [reply]
那希望破站能像 #1464 一样把合集选集列表的这些按钮也给改成 `<a>` 吧。
